test/mocks/network/mocks.h

```diff
@@ -441,6 +441,8 @@ class MockConnectionHandler : public ConnectionHandler {
   MOCK_METHOD(void, enableListeners, ());
   MOCK_METHOD(void, setListenerRejectFraction, (UnitFloat), (override));
   MOCK_METHOD(const std::string&, statPrefix, (), (const));
+
+  std::atomic<uint64_t> num_handler_connections_{};
```

Comment:
Why is this atomic? I don't think any of the mocks are used in an unsynchronized threaded context.

Reply:
Good point! Even the default behavior is not multi-threaded in mock object.
Will fix